english/explore-outside/talk-child/talk-more/talk-more.md

```diff
@@ -0,0 +1,12 @@
+As you continue to walk closer to the child you see that his red eys are from crying.
+He actually looks very sad.  
+
+He says in a thick Pittsburgh accent, "Why did yuns eat the Marshmellow house? That's my house!"
+
+What do you say back?
+
+[Oh sorry, I was trapped in there. How do I get home?](./nice-talk/nice-talk.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.